src/app/admin-panel/admin-panel/students/students.component.html

```diff
@@ -17,11 +19,12 @@
   </ng-template>
   <ng-template pTemplate="header">
     <tr>
-      <th *ngFor="let col of cols">
+      <th *ngFor="let col of cols" [pSortableColumn]="col.field">
         {{col.header}}
+        <p-sortIcon [field]="col.field"></p-sortIcon>
       </th>
       <th>
-        Кнопка
```

Comment:
please remove this row and make edit button appears on hover at the right-hand corner as well as it is on others screens

Reply:
Changed. Button appears on hover.